ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/qos-profiles.j2

```diff
@@ -16,6 +16,8 @@ qos profile {{ profile }}
    tx-queue {{ tx_queue }}
 {%         if qos_profiles[profile].tx_queues[tx_queue].bandwidth_percent is arista.avd.defined %}
       bandwidth percent {{ qos_profiles[profile].tx_queues[tx_queue].bandwidth_percent }}
+{%         elif qos_profiles[profile].tx_queues[tx_queue].bandwidth_guaranteed_percent is arista.avd.defined %}
```

Comment:
Bandwidth guaranteed should be with in qos profile but not inside tx-queue:
```
CL(config-qos-profile-test1)#bandwidth ?
  guaranteed  Set guaranteed bandwidth

```

FYI, with in tx-queue the bandwidth options:
```
CL(config-qos-profile-test1-txq-0)#bandwidth ?
  percent  Set proportion of the total bandwidth when scheduling is round-robin
```

Reply:
this is hardware dependent, which is what the customer hit in this case. So we do need the guaranteed percent under the tx queue on some platforms.

Reply:
Ok. In that case would it be ok to add the same knob under qos profile as well along with adding it to the tx-queue.

Reply:
I don't think we should. I have tried on 7010, 7050X3 and 7020SR and none of them accept the command directly on the profile. I think it is a bug that it is available at that level, since the help text suggests that it should be under the queue:
```
cdv471(config-s-test-qos-profile-test)#?
  abort                  Abandon all changes
  bandwidth              Configure guaranteed/round-robin minimum bw for this queue
...
```